models/BaseAdobe.cfc

```diff
@@ -1670,6 +1671,7 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 		if( !isNull( getMailSignKeyAlias() ) ) { thisConfig.keyAlias = getMailSignKeyAlias(); }
 		if( !isNull( getMailSignKeyPassword() ) ) { thisConfig.keypassword = passwordManager.encryptMailServer( getMailSignKeyPassword() ); }
 		if( !isNull( getMailLogEnabled() ) ) { thisConfig.mailsentloggingenable = getMailLogEnabled() ? true : false; }
+		if( !isNull( getMaintainConnections() ) ) { thisConfig.maintainConnections = getMaintainConnections() ? true : false; }
```

Comment:
You're calling `getMaintainConnections()`, but never created that setting.  See my suggested name in the other comment.  Notice all the mail settings generally start with the word "mail".  Also, if there is a chance the key won't already exist in the user's config XML file, use bracket notation to ensure we create it with the correct case.
```
thisConfig[ 'maintainconnections' ] = ..
```